src/main/scripts/install.sh

```diff
@@ -28,24 +28,44 @@ while getopts "l:u:p:" opt; do
     esac
 done
 
-# Variables
-imKitName=agent.installer.linux.gtk.x86_64_1.9.0.20190715_0328.zip
-repositoryUrl=http://www.ibm.com/software/repositorymanager/com.ibm.websphere.ND.v90
-wasNDTraditional=com.ibm.websphere.ND.v90_9.0.5001.20190828_0616
-ibmJavaSDK=com.ibm.java.jdk.v8_8.0.5040.20190808_0919
+# Wait untile the data disk is partitioned and mounted
+output=$(df -h)
+while echo $output | grep -qv "/datadrive"
+do
+    sleep 10
+    echo "Waiting for data disk partition & moute complete..."
+    output=$(df -h)
+done
+name=$(df -h | grep "/datadrive" | awk '{print $1;}' | grep -Po "(?<=\/dev\/).*")
+echo "UUID=$(blkid | grep -Po "(?<=\/dev\/${name}\: UUID=\")[^\"]*(?=\".*)")   /datadrive   xfs   defaults,nofail   1   2" >> /etc/fstab
+
+# Update applications installed on the system
+yum update -y
+
+# Move tWAS entitlement check and application patch script to /var/lib/cloud/scripts/per-instance
+mv was-check.sh /var/lib/cloud/scripts/per-instance
+
+# Move tWAS installation properties file to /datadrive
+mv virtualimage.properties /datadrive
+
+# Get tWAS installation properties
+source /datadrive/virtualimage.properties
 
 # Create installation directories
-mkdir -p /opt/IBM/InstallationManager/V1.9 && mkdir -p /opt/IBM/WebSphere/ND/V9 && mkdir -p /opt/IBM/IMShared
+mkdir -p ${IM_INSTALL_DIRECTORY} && mkdir -p ${WAS_ND_INSTALL_DIRECTORY} && mkdir -p ${IM_SHARED_DIRECTORY}
 
 # Install IBM Installation Manager
-wget -O "$imKitName" "$imKitLocation" -q
+wget -O "$IM_INSTALL_KIT" "$imKitLocation" -q
```

Comment:
What value are you using for imKitLocation? I think this should be added to virtualimage.properties as well if this param is needed. We should get the latest version from https://public.dhe.ibm.com/ibmdl/export/pub/software/im/zips/agent.installer.linux.gtk.x86_64.zip

Reply:
Previously I think user can't download installation manager kit until he/she signs in with her/his IBMid. So I downloaded it manually and then upload to Azure storage account. The value of `imKitLocation` is the url of installation manager kit in Azure storage blob. 

Since now it looks like IM kit can be downloaded via a public url, it will simplify the process and it can be added to virtualimage.properties as you suggested above.

Reply:
Fixed in commit [2c7ee2f](https://github.com/WASdev/azure.websphere-traditional.image/commit/2c7ee2fb76d6637fbf6fb9e13be44697ab886519#diff-5428e4667ed75183e8c0e37d59551ac0eaf5d836244d475d7aaf58c6422253dcR18-R19).